worlds/mm-recomp/Rules.py

```diff
@@ -296,14 +330,191 @@ def get_location_rules(player):
             lambda state: state.has("Progressive Bow", player) and (state.has("Boss Key (Woodfall)", player) or state.has("Odolwa's Remains", player)),
         "Woodfall Temple Odolwa's Remains":
             lambda state: state.has("Progressive Bow", player) and (state.has("Boss Key (Woodfall)", player) or state.has("Odolwa's Remains", player)),
-
+            
+            
+        "Mountain Village Darmani":
+            lambda state: state.has("Lens of Truth", state, player) and state.has("Progressive Magic Upgrade", player) and can_play_song("Song of Healing", state, player),
+        "Mountain Village Hungry Goron":
+            lambda state: state.has("Goron Mask", player) and (can_use_fire_arrows(state, player) or (state.has("Deku Stick", state, player) and state.has("Progressive Magic Upgrade", player))),
```

Comment:
This needs to be changed.
```suggestion
            lambda state: state.has("Goron Mask", player) and state.has("Progressive Magic Upgrade", player) and (can_play_song("Goron's Lullaby", state, player) and state.has("Deku Stick", player) or can_use_fire_arrows(state, player)),
```

Reply:
Deku Stick isn't an item, so this will currently never be true.